tests/std/tests/Dev11_0535636_functional_overhaul/test.cpp

```diff
@@ -2195,56 +2221,56 @@ struct BoolWrapper {
 struct TestNotFn {
     int m_x;
 
-    explicit TestNotFn(const int x) : m_x(x) {}
+    explicit constexpr TestNotFn(const int x) : m_x(x) {}

```

Comment:
There are occurrences of `explicit constexpr` in the test that should follow the `constexpr explicit` convention.

Reply:
Done